src/Network/Network/help/Add-AzLoadBalancerRuleConfig.md

```diff
@@ -47,6 +63,20 @@ The first command gets the load balancer named MyLoadBalancer, and then stores i
 The second command uses the pipeline operator to pass the load balancer in $slb to **Add-AzLoadBalancerRuleConfig**, which adds the rule configuration named NewRule.
 The third command will update the load balancer in azure with the new Load Balancer Rule Config.
 
+### Example 2: Add a rule configuration with two backend address pools to a load balancer
+```
+PS C:\>$slb = Get-AzLoadBalancer -Name "MyLoadBalancer" -ResourceGroupName "MyResourceGroup"
+PS C:\> $MyBackendPool1 = Get-AzLoadBalancerBackendAddressPool -ResourceGroupName $resourceGroup -LoadBalancerName $MyLoadBalancer -Name $backendPool1Name
+PS C:\> $MyBackendPool2 = Get-AzLoadBalancerBackendAddressPool -ResourceGroupName $resourceGroup -LoadBalancerName $MyLoadBalancer -Name $backendPool2Name
+PS C:\> $slb | Add-AzLoadBalancerRuleConfig -Name "NewRule" -FrontendIPConfiguration $slb.FrontendIpConfigurations[0] -Protocol "All" -FrontendPort 0 -BackendPort 0 -BackendAddressPools $MyBackendPool1, $MyBackendPool2
+PS C:\>$slb | Set-AzLoadBalancer
+```
+This scenario is used for Gateway Load Balancer
```

Comment:
Is this okay?
also should I add descriptions like this in other places?

Reply:
This enables Gateway Load Balancer to have multiple backend pools